activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Add `ActiveRecord::Relation#in_batches` to work with records and relations
+    in batches.
+
+    Available options are `of` (batch size), `load`, `begin_at`, and `end_at`.
+
+    Examples:
+
+        Person.in_batches.each_record(&:party_all_night!)
+        Person.in_batches.update_all(awesome: true)
+        Person.in_batches(of: 100).delete_all
+        Person.in_batches.map { |relation| relation.to_sql }
```

Comment:
Can we find a better example than `to_sql`? That's generally not something people should use.